From ba9735f71e72cd3c7b57b1558a69c9f111316f65 Mon Sep 17 00:00:00 2001
From: Mehdi Sabwat <mehdisabwat@gmail.com>
Date: Tue, 31 Dec 2019 12:02:20 +0100
Subject: [PATCH 1/1] contrib: delete empty variable

As a fwp on c29409d1a742e65b6b2f3c95702196ff9ab1570c this commit fixes an issue
on platforms that are not listed.
---
 contrib/src/main.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/contrib/src/main.mak b/contrib/src/main.mak
index 897020517e..a0469fb3e0 100644
--- a/contrib/src/main.mak
+++ b/contrib/src/main.mak
@@ -646,7 +646,6 @@ ifdef HAVE_CROSS_COMPILE
 	echo "set(PKG_CONFIG_EXECUTABLE $(PKG_CONFIG))" >> $@
 endif
 
-MESON_SYSTEM_NAME =
 ifdef HAVE_WIN32
 	MESON_SYSTEM_NAME = windows
 else
-- 
2.24.1

